code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -44,3 +44,242 @@
 /mob/living/simple_animal/hostile/retaliate/clown/attack_hand(mob/living/carbon/human/M)
 	..()
 	playsound(src.loc, 'sound/items/bikehorn.ogg', 50, 1)
+
+/mob/living/simple_animal/hostile/retaliate/clown/fleshclown
+	name = "Fleshclown"
+	desc = "A being forged out of the pure essence of pranking, cursed into existence by a cruel maker."
+	icon = 'icons/mob/clown_mobs.dmi'

```

Comment:
You're repeating a lot of variables for each of these types, use a shared parent to reduce that a bit

Reply:
Alright. I'll reduce that in the next commit, as well as banana improvements

Reply:
I was already able to cut it down a little just by reverting a couple values to match the original clown, though with a little effort I can make an additional parent

Reply:
I'm going to make some of them have clownhulk as a parent and make some of the others closer to regular clown

Reply:
I fixed a lot of the redundant variables. I can do more if that's not enough